dev/partkey.md

```diff
@@ -172,3 +172,41 @@ wrapped in a $\Credential$ struct, containing the following fields:
 
 And this verified credential is wrapped in a $\Vote$ struct with _Raw Vote_ 
 ($\mathrm{R}$), _Verified Credential_ ($\Credential$), and _Signature_ ($\Sig$).
+
+
+## Algorand State Proof Keys 
+### Algorand's Committable Ephemeral Keys Scheme - Merkle Keystore
+
+Algorand archives [forward security](https://en.wikipedia.org/wiki/Forward_secrecy) using the Merkle Keystore scheme. This scheme consists of designate an ephemeral key for each round in which it will be used. The scheme uses a Merkle tree to generate a commitment on those keys. 
+The private key must be deleted be the done in order the completely archives forward secrecy.
+
+The ephemeral key can be generated using one of Algorand's supported digital signature algorithms.
+
+
+In order to bound the proofs on this tree, the tree's depth is bound to 16. Hence, the maximum number of keys which will be created is 2^16.
+
+#### Public Commitment
+
+The scheme generate keys that correspond to a participation period. Given _FirstValidRound_, _LastValidRound_ and an _Interval_, a key is generated for each _Round_ that holds:\newline
+ _FirstValidRound_ $\leq$  _Round_ $\leq$ _LastValidRound_ and  _Round_  % _Interval_  = 0
+
+Note that there might not be a commitment for certain period (i.e does not include any round that holds _Round_  % _Interval_  = 0) and the value of 0 represents an empty commitment 
```

Comment:
```suggestion
Note that there might not be a commitment for certain period (i.e does not include any round such that _Round_  % _Interval_  = 0) and the value of 0 represents an empty commitment 
```

What is the purpose of excluding such rounds?

Reply:
My intention was to state that 
1 the value 0 represents an empty commitment.
2 user might get an empty commitment if he tries to generate keys for short period (i.e: a period that doesn't include a round such that: round % interval ==0.)

did that make sense?

Reply:
I see